bundles/ruby/ruby_mode.moon

```diff
@@ -68,7 +68,7 @@ continuation_indent = (line, indent_level) ->
   code_blocks:
     multiline: {
       { r'\\s+do(?:\\s*\\|[^|]+\\|)?\\s*$', '^%s*end', 'end' },
-      { r'^\\s*def\\s+\\w[\\w.\\d]+(?:\\s*\\([^)]*\\))?\\s*$', '^%s*end', 'end' },
+      { r'^\\s*def\\s+\\w[\\w.\\d?!]+(?:\\s*\\([^)]*\\))?\\s*$', '^%s*end', 'end' },
```

Comment:
@nilnor I added specs. However, I can't test them right now, since my Lua installation is seriously messed up, so... \*crosses fingers*